### PR TITLE
Webinar teaser

### LINKED ISSUE #339: Implement "Teaser (Webinar)" block
Sample: https://main--moleculardevices--hlxsites.hlx.page/technology/electrophysiology
Reference: https://www.moleculardevices.com/technology/electrophysiology

### PR DESCRIPTION
**Pull request by lydiapuric**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #339

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/technology/electrophysiology
- After: https://webinar-teaser--moleculardevices--hlxsites.hlx.page/technology/electrophysiology

### CODE REVIEW

File: blocks/webinar-teaser/webinar-teaser.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  const videoSection = nodes[4];
+  videoSection.querySelector('a').setAttribute('target', '_blank');
+  const webinarDescription = nodes[5];
+  const registerButton = nodes[6];
+  const registerButtonLink = registerButton.querySelector('a');
+  registerButtonLink.append(span({ class: 'icon icon-fa-external-link' }));
+  registerButtonLink.setAttribute('target', '_blank');
+
+  block.innerHTML = '';
+  const header = (
```
Comment (mhaack) [inline blocks/webinar-teaser/webinar-teaser.js:22]: We remove the DOM and create an new one for this block. I think it is more effective and elegant to just style what is rendered.

File: blocks/webinar-teaser/webinar-teaser.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import { decorateIcons } from '../../scripts/lib-franklin.js';
+import {
+  div, span,
+} from '../../scripts/dom-helpers.js';
+
+export default function decorate(block) {
+  const root = block.querySelector(':scope > div > div');
+  const nodes = root.children;
```
Comment (mhaack) [inline blocks/webinar-teaser/webinar-teaser.js:8]: Depending on the node order is easy to break for example if there is no speaker title and just the name. We should either style what is already there or work with different heading size , ‚em etc.

**Review by mhaack — APPROVED** [review]
LGTM

File: blocks/webinar-teaser/webinar-teaser.css [hunk 3]
```diff
@@ -0,0 +1,117 @@
+.webinar-teaser {
+  background-image: url('/images/wave-background-techtip-blue-green.png');
```
Comment (mhaack) [inline blocks/webinar-teaser/webinar-teaser.css:2]: You added both file formats for the BG image but here you only use one

Comment (lydiapuric) [reply blocks/webinar-teaser/webinar-teaser.css]: done.

**Pull request merged**